sf_stats.Api/sf_stats.Domain/Entities/Player.cs

```diff
@@ -16,5 +13,10 @@ public class Player : DbEntity
         public int Height { get; set; }
         public int Weight { get; set; }
         public string Grade { get; set; }
+
+        public virtual List<PlayerStat> PlayerStats { get; set; }
+
+        //Feels like this should be a list of TeamSeasons
+        public virtual List<TeamSeasonPlayer> TeamSeasonPlayers { get; set; }
```

Comment:
I am struggling to follow in my head. 

Guessing on player I would want to know get a few things

- What teams they are currently on
- What teams they have been on
- What stats do they have

Guess it depends on the UI. Some of this stuff will be annoying to change later but if it has to be done so be it

Reply:
We should take second tomorrow and go through this on a call. The way it's currently setup, there's no way to tell which team the player scored the stats for, but all the ideas I have for solving that convolute things.